inst/builtin/bs5/shiny/_rules.scss

```
@@ -189,3 +189,72 @@ $icon-classes: ("bi", "fa", "fas", "far", "fab", "material-icons") !default;
     --bslib-color-fg: #008558; // Contrast 4.67
   }
 }
+
+// Modals
+.modal-body, .modal-footer {
+  padding: 1.5rem;
+}
+.modal-header .btn-close {
+  align-self: start;
+  display: flex;
+}
+#shiny-modal-wrapper:has( ~ .modal-backdrop) .modal {
+  // Blur the background when the modal has a backdrop
+  backdrop-filter: blur(2px);
+}
+
+// Progress Bars & Notifications
+// FIXME: Use css layers (dynamic themeing issue)
```

The issue here more about CSS order rather than dynamic theming?

```suggestion
// FIXME: Use css layers
```